test/aws/rails/middleware/ebs_sqs_active_job_middleware_test.rb

```diff
@@ -76,6 +76,46 @@ module Rails
         expect(response[0]).to eq(500)
       end
 
+      it 'successfully invokes job when docker default gateway ip is changed' do
+        mock_rack_env = create_mock_env('192.168.176.1', 'aws-sqsd/1.1', false)
+        test_middleware = EbsSqsActiveJobMiddleware.new(mock_rack_app)
+
+        proc_net_route = <<~CONTENT
+          Iface\tDestination\tGateway\tFlags\tRefCnt\tUse\tMetric\tMask\tMTU\tWindow\tIRTT
+          eth0\t00000000\t01B0A8C0\t0003\t0\t0\t0\t00000000\t0\t0\t0
+          eth0\t00B0A8C0\t00000000\t0001\t0\t0\t0\t00F0FFFF\t0\t0\t0
+        CONTENT
+
+        allow(File).to receive(:exist?).and_call_original
+        allow(File).to receive(:open).and_call_original
+
+        expect(File).to receive(:exist?).with('/proc/net/route').and_return(true)
+        expect(File).to receive(:open).with('/proc/net/route').and_return(StringIO.new(proc_net_route))
+        expect(test_middleware).to receive(:app_runs_in_docker_container?).and_return(true)
+
+        response = test_middleware.call(mock_rack_env)
+
+        expect(response[0]).to eq(200)
+        expect(response[1]['Content-Type']).to eq('text/plain')
+        expect(response[2]).to eq(['Successfully ran job ElasticBeanstalkJob.'])
+      end
+
+      it 'successfully invokes job when /proc/net/route is not exist' do
```

Comment:
Wording nit, `is not` -> `does not`

Reply:
Thanks.
Fixed it.

https://github.com/aws/aws-sdk-rails/pull/99/commits/9efd2755c118865fd8e18e143f99757a6c554836